primitives/sidechain-mc-hash/src/lib.rs

```diff
@@ -119,6 +121,10 @@ impl McHashInherentDataProvider {
 	pub fn mc_block(&self) -> McBlockNumber {
 		self.mc_block.number
 	}
+
+	pub fn mc_hash(&self) -> McBlockHash {
+		self.mc_block.hash.clone()
+	}
```

Comment:
Help my understand: how was `McBlockHash` extracted from  `McHashInherentDataProvider` before? I'm surprised that we accessed hash in a different way or we didn't at all.